numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -4256,7 +4256,7 @@ def add_newdoc(place, name, doc):
 
 add_newdoc('numpy._core.umath', 'tanh',
     """
-    Compute hyperbolic tangent element-wise.
+    Hyperbolic tangent, element-wise.
```

Comment:
I guess this is what motivated the PR?

If we are already changing this, I think it should be `Element-wise hyperbolic tangent`. What do you think? Whatever we decide, there are many other appearances of this in the file. It would be nice to align them all to the same template.

Reply:
yes, that's what motivated the PR. I am ok with "Element-wise hyperbolic tangent". There are lots of cases like this in ufunc_docstrings.py. Should we start doing in batches, let's say "Trigonometric functions" and "Hyperbolic functions" first?

Reply:
Sounds good to progress in batches.